src/screen.c

```diff
@@ -4828,6 +4836,28 @@ win_line(
 		    }
 		    else
 			mb_utf8 = FALSE;
+#endif
+		}
+
+		if (leadcol != 0 && ptr <= line + leadcol && c == ' ')
```

Comment:
Here, it looks like a line full of spaces is seen as leading spaces. Is this as intended?

It would be wise to write tests for these changes, that way you would see if your (or my!) assumptions about code are correct or incorrect...

Reply:
That is a really good observation. It raises the question: Should (assuming that lcs_trail and lcs_lead are both set) a line of spaces be treated as *trailing* or *leading* spaces?

I didn't really think about this. In all honesty, I don't see a reason to prefer one over the other.